app/views/appeal/discussion.scala

```diff
@@ -97,7 +97,7 @@ object discussion {
       ),
       if (modData.isEmpty)
       h2(
-        renderMark(ctx.me.get)
+        ctx.me map renderMark
```

Comment:
It's better but still weird. If there's no `ctx.me`, then there's no `h2`?

You may tell me that there will always be a `User`. Then, prove it by passing it to `renderAppeal` as an argument.